share/zfsnap/commands/send.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+
+# This file is licensed under the BSD-3-Clause license.
+# See the AUTHORS and LICENSE files for more information.
+
+SSH_KEY=~/.ssh/id_rsa
+#EMAIL=none@none
+user=$USER
+LOCAL='false'
+FULL='false'
+REVERSE='false'
+
+# FUNCTIONS
+Help() {
+    cat << EOF
+${0##*/} v${VERSION}
+
+Syntax:
+${0##*/} destroy [ options ] zpool/filesystem ...
```

Comment:
What is the intended syntax? That'll make it a bit easier to review some of the logic/help options.

what do you think of `zfsnap send [options] zpool/filesystem [[user@]host]:zpool/filesystem`?